lib/EnzymeCore/Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "EnzymeCore"
 uuid = "f151be2c-9106-41f4-ab19-57ee4f262869"
 authors = ["William Moses <wmoses@mit.edu>", "Valentin Churavy <vchuravy@mit.edu>"]
-version = "0.8.9"
+version = "0.8.10"
```

Comment:
so one caveat here, can you grep through the code for places with BatchDuplicated and check that we add the requisite condition for StackedBatchDuplicated.

Even more ideally we would have some interface function is_batched or something that can be used by either

Reply:
Active{Float64} -> Active